.circleci/real_config.yml

```diff
@@ -2840,7 +2840,7 @@ jobs:
                   PROJECT=$(terraform -chdir=tools/slurm/terraform output --raw project)
                   gcloud compute scp agent/build/determined-agent "$INSTANCE_NAME":~ --zone $ZONE
                   gcloud compute ssh --zone "$ZONE" "$INSTANCE_NAME" --project "$PROJECT" -- \
-                  srun determined-agent --master-host=<<parameters.master-host>> --master-port=<<parameters.master-port>> --resource-pool=default --container-runtime=<<parameters.container-run-type>>
+                  srun determined-agent --master-host=<<parameters.master-host>> --master-port=<<parameters.master-port>> --resource-pool=default
```

Comment:
We can delete the ```agent-use``` parameter and all steps that use it since it should be unused

So I would completely delete this ```when``` block and the ```when``` block below this

Reply:
Yes I understand your comment here and will update it.
We would need to remove the  `test-e2e-slurm` case as well, right?

```
jobs:
  test-e2e-slurm:
      parameters:
        - when:
                  condition:
                    equal: ["-A", <<parameters.agent-use>>]
```